js/package.json

```diff
@@ -40,11 +40,18 @@
     "@jupyterlab/application": "^3.0.0",
     "@jupyterlab/apputils": "^3.0.0",
     "@jupyterlab/coreutils": "^5.0.0",
+    "@jupyterlab/launcher": "^3.0.2",
     "@lumino/disposable": "^1.1.2",
-    "requests-helper": "^0.1.5"
+    "@types/node": "^14.14.22",
+    "axios": "^0.21.1",
+    "image-pixels": "^2.2.2",
```

Comment:
remove dep

Reply:
needed to convert png/jpg icons to svg's jupyterlab can display in launcher

Reply:
nope, you can use `kernelIconUrl`: https://github.com/jupyterlab/jupyterlab/blob/60336e5c8d600f609771a78fbeedb50c5dc070e5/packages/launcher/src/index.tsx#L357

I use a `png` in this library (as an example): https://github.com/jpmorganchase/jupyterlab_templates/blob/main/js/src/index.ts#L175

Reply:
Cool, that would have saved me some time 🙃

Reply:
@ofekengel yeah its not documented anywhere I don't think...it took a lot of digging to figure out how to make a "fake" kernel launcher :joy: